1036-timeline-design-guidance.md

```diff
@@ -0,0 +1,47 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-22
+---
+
+# Timeline Component
+
+The [timeline](https://clarity.design/documentation/timeline) component is a visual indicator that shows progress through a workflow. It can also guide the user through specific processes or pre-defined steps.
+
+## Guidance
+
+### Timeline Usage
+
+- **Must** use a timeline when you need to showcase a list of events that are relevant to the user in chronological order.
+- **Must** use a timeline when users need to have an overview of a workflow and where they are in the process.
+- **Must** use a timeline when you need to display more complex content in each step.
+- **Must** use a timeline when you need more space for longer forms with a fewer number of steps. For many steps with small data set in each step use [stepper](https://clarity.design/documentation/stepper).
+- **May** use a timeline when you need to break down a complex and long work processes into multiple concrete steps (more than two steps).
+
+For more guidance on multi-step process workflows, see the [Multi-Step Workflow Pattern](https://clarity.design/documentation/multi-step-workflow).
+
+### Timeline Layouts
+
+There are two types of layouts, `horizontal` and `vertical`.
```

Comment:
```suggestion
There are two types of layouts: horizontal and vertical.
```

Inline code isn't necessary for this. This is not code documentation. Please update all.